ERCS/erc-7515.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: 7515
+title: NFT Alternative Text in Metadata
+description: Populate alternative text field in NFT Metadata with image description for screen reader.
+author: Mona Rassouli (@DecoratedWings)
+discussions-to: https://ethereum-magicians.org/t/proposal-for-informational-eip-simple-accessibility-recommendation/14639
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-08-15
+---
+
+## Abstract
+
+This ERC advocates for the consistent inclusion of alternative text in NFT metadata. Marketplaces, creators, and decentralized applications should adopt this best practice to improve accessibility. When rendering NFT images on frontend interfaces, these platforms can leverage the `alt` attribute effectively.
+
+## Motivation
+
+Alternative text assists different users who leverage assistive technology. Users who leverage screen readers for instance, benefit from an accurate, concise description of the image.
+
+Given that NFTs are portable between marketplaces, websites, and dapps, it is more efficient to create a suitable description upon creation (minting) of the NFT. In this way, the description can be attached to the image and ported between interfaces. The alternative text only needs to be written one time, eliminating the need to recreate it from scratch on every interface.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+The `alt` attribute should be positioned at the root level within the metadata structure. It MUST provide a concise and accurate description of the image.
+
```

Comment:
I'd like to see a JSON Schema that can be [`allOf`](https://json-schema.org/understanding-json-schema/reference/combining#allOf)'d with base schema, like from [ERC-721](https://eips.ethereum.org/EIPS/eip-721).